src/install/prepare-sources-tarball.sh

```diff
@@ -2,7 +2,7 @@
 tmp=$(mktemp -d)
 VERSION=$(cat ./src/cli/cli.go | grep 'Version = "v' | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
 echo $VERSION
-git clone --depth 1 https://github.com/schollz/croc $tmp/croc
-(cd $tmp/croc && go mod tidy && go mod vendor)
-(cd $tmp && tar -cvzf croc_${VERSION}_src.tar.gz croc)
+git clone --depth 1 https://github.com/schollz/croc $tmp/croc-${VERSION}
```

Comment:
Does this hook run _after_ you tag the repo? If so, it would be better to change this to `git clone -b v${VERSION} --depth 1 https://github.com/schollz/croc $tmp/croc-${VERSION}`, to make sure this script clones the right thing (in case you decide to push new commits seconds after releasing a new version)

Reply:
Yeah it does run the hook after tagging and pushing! This is a good point. It should probably do a `git clone` and then a `git checkout` in the dir.

Reply:
The `-b` flag for `git clone` is exactly doing the `git checkout` part, but in one command 😉  I'll update this PR in a second.